README.md

```diff
@@ -1,56 +1,116 @@
-   - Improve management efficiency and reduce costs without compromising performance and availability
-
-## Quick start
-
-See [Quick start](https://open.oceanbase.com/quickStart) to try out OceanBase Database.
-
-## System architecture
-
+<p align="center">
+    <a href="https://github.com/oceanbase/oceanbase">
+        <img alt="OceanBase Logo" src="images/logo.svg" width="50%" />
+    </a>
+</p>
+<p align="center">
+    <a href="https://github.com/oceanbase/oceanbase/blob/master/LICENSE">
+        <img alt="license" src="https://img.shields.io/badge/license-MulanPubL--2.0-blue" />
+    </a>
+    <a href="https://github.com/oceanbase/oceanbase/releases">
+        <img alt="license" src="https://img.shields.io/badge/dynamic/json?color=blue&label=release&query=tag_name&url=https%3A%2F%2Fapi.github.com%2Frepos%2Foceanbase%2Foceanbase%2Freleases%2Flatest" />
+    </a>
+    <a href="https://github.com/oceanbase/oceanbase">
+        <img alt="stars" src="https://img.shields.io/badge/dynamic/json?color=blue&label=stars&query=stargazers_count&url=https%3A%2F%2Fapi.github.com%2Frepos%2Foceanbase%2Foceanbase" />
+    </a>
+    <a href="https://github.com/oceanbase/oceanbase">
+        <img alt="forks" src="https://img.shields.io/badge/dynamic/json?color=blue&label=forks&query=forks&url=https%3A%2F%2Fapi.github.com%2Frepos%2Foceanbase%2Foceanbase" />
+    </a>
+    <a href="https://en.oceanbase.com/docs/community-observer-en-10000000000829617">
+        <img alt="English doc" src="https://img.shields.io/badge/docs-English-blue" />
+    </a>
+    <a href="https://www.oceanbase.com/docs/oceanbase-database-cn">
+        <img alt="Chinese doc" src="https://img.shields.io/badge/文档-简体中文-blue" />
+    </a>
+    <a href="https://github.com/oceanbase/oceanbase/actions/workflows/compile.yml">
+        <img alt="building status" src="https://img.shields.io/github/actions/workflow/status/oceanbase/oceanbase/compile.yml?branch=master" />
+    </a>
+    <a href="https://github.com/oceanbase/oceanbase/commits/master">
+        <img alt="last commit" src="https://img.shields.io/github/last-commit/oceanbase/oceanbase/master" />
+    </a>
+    <a href="https://join.slack.com/t/oceanbase/shared_invite/zt-1e25oz3ol-lJ6YNqPHaKwY_mhhioyEuw">
+        <img alt="Join Slack" src="https://img.shields.io/badge/slack-Join%20Oceanbase-brightgreen?logo=slack" />
+    </a>
+</p>
+
+**OceanBase Database** is a distributed relational database. It is developed entirely by Ant Group. OceanBase Database is built on a common server cluster. Based on the [Paxos](https://lamport.azurewebsites.net/pubs/lamport-paxos.pdf) protocol and its distributed structure, OceanBase Database provides high availability and linear scalability. OceanBase Database is not dependent on specific hardware architectures.
+
+# Key features
+* Transparent Scalability
+* Ultra-fast Performance
+* Real-time Operational Analytics
+* Continuous Availability
+* MySQL Compatible
+* Cost Effeciency
```

Comment:
For every key features, please add some details, please copy the explanation from website.

Reply:
I worry about that this part will be too much after adding details. But if using detail tabs, it seems that it does not conform to user habits. what about your suggestion? @tisonkun @caifeizhi

Reply:
In my opinion, it would be appropriate to provide a link for each key feature, but currently it seems there is no way to directly link to each key feature. Therefore, an alternative approach may be to add a brief description after each key feature.